graph/config/cytoscape/cytoscape.go

```diff
@@ -71,6 +71,7 @@ type NodeData struct {
 	HasCB           bool                `json:"hasCB,omitempty"`           // true (has circuit breaker) | false
 	HasMissingSC    bool                `json:"hasMissingSC,omitempty"`    // true (has missing sidecar) | false
 	HasVS           bool                `json:"hasVS,omitempty"`           // true (has route rule) | false
+	HasHealthConfig map[string]string   `json:"HasHealthConfig,omitempty"` // set to the health config override
```

Comment:
three things:
- json should be `hasHealthConfig`
- maintain alphebetical (move up after HasCB)
- recommended: I still think map[string]string deserves to be typed.  I had suggested `HealthConfig`, you suggest ` []Tolerance`.  But for me `[]Tolerance` introduces another term that then needs definition.    Remember that the type declared here is just local to this Cytoscape config generator, declared above with the other types.  So I think something like this would be fine (update the comment as you see fit):
```
// HealthConfig is a flexible map to pass available health information, tolerances, etc, to be interpreted by the client.
type HealthConfig map[string][string]
```

Reply:
understood. Thanks